dust_extinction/dust_extinction.py

```diff
@@ -1228,6 +1228,136 @@ def evaluate(self, in_x, Rv):
                                  optnir_axav_x, optnir_axebv_y/Rv,
                                  self.x_range, 'F99')
 
+class F99FM07(BaseExtRvModel):
+    """
+    F99 extinction model calculation
+
+    Updated with the NIR Rv dependence in
```

Comment:
Ok.  I see this info now.  Maybe just call this FM07 or something?  F99FM07 does not capture all the info (e.g., FM04).  Is everything in FM07 in this code?  If so, maybe the name could be FM07 and the documentation explain the details?